grafana/resource_library_panel.go

```diff
@@ -162,6 +172,18 @@ func ReadLibraryPanel(ctx context.Context, d *schema.ResourceData, meta interfac
 	d.Set("created", panel.Meta.Created.String())
 	d.Set("updated", panel.Meta.Updated.String())
 
+	connections, err := client.LibraryPanelConnections(uid)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	dashboardIds := []int64{}
```

Comment:
Small performance tip: given that we know the final length of this slice, please use the following:
```suggestion
	dashboardIds := make)[]int64, 0, len(*connections))
```

This will reduce the number of allocations Go has to make to create this slice.

Reply:
:) https://github.com/grafana/terraform-provider-grafana/pull/370/commits/7e5c9c247ea5059268d153beb3f3f33eb73311a1